packages/sdk/src/doc/gen/common.ts

```diff
@@ -98,6 +98,10 @@ export interface UssItem {
    * Whether the item is a directory
    */
   isDir: boolean;
```

Comment:
Do we want to keep `isDir`? It seems a bit redundant if this information is contained in `mode`

Reply:
> Without isDir, the consumer needs to use a startsWith check to determine if the entry is a directory

I like the convenience of not having to do a `startsWith`. 😋 

> Something else to consider is that this also gracefully handles symbolic links to directories, and without the isDir boolean, it won't be obvious from the client that the path is a directory without following the link to its target. 

Trying to understand this part though...
After doing some testing, I still get that `isDir: false` for a symlink to a directory 🤔

Reply:
Addressed in a9a5da7